src/apps/answers/service.py

```diff
@@ -219,6 +219,46 @@ async def _validate_applet_for_user_response(self, applet_id: uuid.UUID) -> None
         if not roles:
             raise UserDoesNotHavePermissionError()
 
+    async def _validate_temp_take_now_relation_between_subjects(
+        self, respondent_subject_id: uuid.UUID, source_subject_id: uuid.UUID, target_subject_id: uuid.UUID
+    ) -> None:
+        relation_respondent_source = await SubjectsCrud(self.session).get_relation(
+            respondent_subject_id, source_subject_id
+        )
+        relation_respondent_target = await SubjectsCrud(self.session).get_relation(
```

Comment:
please, query database just before use it, so don't query for second relation if first one fail